python/lsst/pipe/supertask/activator.py

```diff
@@ -131,6 +132,34 @@ def activate(self):
         """
         pass
 
+def get_inheritance(stask_class, subclasses=[]):
```

Comment:
That is the idiomatic way to do it (removing the spaces around the `=` of course). Otherwise if you ever use the default keyword argument you will get the same list every single time you call it. The list is allocated at compile time. See:
```
>>> def test(arg=[]):
...    arg.append("b")
...    return arg
... 
>>> test()
['b']
>>> test()
['b', 'b']
```